gramps/gen/filters/rules/family/_fatherhasidof.py

```diff
@@ -48,5 +56,13 @@ class FatherHasIdOf(RegExpIdBase):
     name = _("Families having father with Id containing <text>")
     description = _("Matches families whose father has a specified " "Gramps ID")
     category = _("Father filters")
-    base_class = RegExpIdBase
-    apply = father_base
+
+    def apply_to_one(self, db: Database, family: Family) -> bool:  # type: ignore[override]
+        father_handle = family.father_handle
+        if father_handle:
+            father = db.get_person_from_handle(father_handle)
+            if father:
+                return super().apply_to_one(db, father)
+            else:
+                return False
```

Comment:
```suggestion
            return super().apply_to_one(db, father) if father else False
```
Question: is the shorter ternary style or move verbose if: else: preferred in gramps?

Reply:
I think it depends on the situation and clarity. But makes sense for this one.